src/test/java/com/yelp/nrtsearch/server/grpc/ReplicationServerTest.java

```diff
@@ -372,7 +309,7 @@ public void testSyncOnIndexStart() throws IOException, InterruptedException {
                     .setTopHits(10)
                     .addAllRetrieveFields(LuceneServerTest.RETRIEVED_VALUES)
                     .build());
-    assertEquals(0, searchResponseSecondary.getHitsCount());
+    assertEquals(4, searchResponseSecondary.getHitsCount());
```

Comment:
I think these tests expect syncInitialNrtPoint to be false at startup. It would be good to retain the behavior of the existing tests